recipes/pyfes/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "pyfes" %}
+{% set version = "2024.6.0" %}
+{% set sha256 = "8e89ff5bb318f7131471f52233602fdc5ab4bcdc3293f64d8314a6db45d1acce" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CNES/aviso-fes/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: {{ build }}
```

We avoid abusing jinjas templates, specially for single use variables, b/c it obscures the recipe and make it harder for the bot to issue auto-updates.  

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/CNES/aviso-fes/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
  sha256: 8e89ff5bb318f7131471f52233602fdc5ab4bcdc3293f64d8314a6db45d1acce

build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
```